### PR TITLE
#66 Add reading level to StoryBook converter

### PR DESCRIPTION
**Pull request by gscdev**

### CODE REVIEW

File: content-provider-utils/src/main/java/ai/elimu/content_provider/utils/converter/CursorToStoryBookGsonConverter.java [hunk 1]
```diff
@@ -33,11 +34,17 @@ public static StoryBookGson getStoryBookGson(Cursor cursor) {
         ImageGson coverImage = new ImageGson();
         coverImage.setId(coverImageId);
 
+        int columnReadingLevel = cursor.getColumnIndex("readingLevel");
+        String readingLevelName = cursor.getString(columnReadingLevel);
+        ReadingLevel readingLevel = ReadingLevel.valueOf(readingLevelName);
+        Log.i(CursorToStoryBookGsonConverter.class.getName(), "readingLevel: " + readingLevelName);
```
Comment (nya-elimuai) [inline content-provider-utils/src/main/java/ai/elimu/content_provider/utils/converter/CursorToStoryBookGsonConverter.java:40]:
```suggestion
        Log.i(CursorToStoryBookGsonConverter.class.getName(), "readingLevel: " + readingLevel);
```

**Review by nya-elimuai — APPROVED** [review]

**Comment by nya-elimuai** [comment]
@gscdev Remind me to release a new version of the `content-provider-utils` library once you merge this pull request 😉

**Review by jo-elimuai — APPROVED** [review]

**Review by nya-elimuai — APPROVED** [review]

**Comment by gscdev** [comment]
@nya-elimuai you can release a new version of `content-provider-utils`

**Comment by nya-elimuai** [comment]
> @nya-elimuai you can release a new version of `content-provider-utils`

@gscdev Done: https://bintray.com/elimu-ai/maven/content-provider-utils

**Pull request merged**